Core/src/main/java/com/plotsquared/core/plot/flag/types/BlockTypeWrapper.java

```diff
@@ -86,15 +97,45 @@ public boolean accepts(final BlockType blockType) {
         }
     }
 
+    public BlockCategory getBlockCategory() {
+        if (this.blockCategory == null && this.blockCategoryId != null) { // only if name is available
+            this.blockCategory = BlockCategory.REGISTRY.get(this.blockCategoryId);
+            if (this.blockCategory == null) {
```

Comment:
if (category is null and registry is not empty), this means that in the case that this method is called before the registry has been loaded, it'll have to refresh next time